docs/reference/changelog.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [0.9.2] - 2019-10-25
+
+### Documentation
+* Update example file [#146](https://github.com/open-contracting/infrastructure/pull/159/commits/cf162cf2e24f3c0cd4489b9a7ff2089c0e11dd3f) - add 'publicAuthority' role and [#136](https://github.com/open-contracting/infrastructure/pull/159/commits/3b58a49139a72b56e469a8eefa2e688ce68b17fd) -   project identifier prefix.
```

Comment:
Please put issue numbers at the start of the bullets and make them links to the issue, rather than the commit. You can also group similar bullets, e.g.

```
* Updates to worked example:
  * [#146](https://github.com/open-contracting/infrastructure/issues/146) - add missing 'publicAuthority' role
  * [#136](https://github.com/open-contracting/infrastructure/issues/136) - add missing project identifier prefix
```

Please update the other bullets too.

Thanks!